code/modules/surgery/bodyparts/parts.dm

```diff
@@ -87,6 +87,48 @@
 	unarmed_damage_high = 10
 	unarmed_stun_threshold = 10
 	body_zone = BODY_ZONE_L_ARM
+	/// Offsets for held items. If your hand is shorter and you wanna move the items up.
+	var/held_offset = 0
+	/// Are we covered with blood?
+	var/bloody = FALSE
+	/// Icon_state for the bloody hand.
+	var/bloody_icon
+
+/obj/item/bodypart/arm/update_limb(dropping_limb, is_creating)
+	. = ..()
+	var/mob/living/carbon/human/human_owner = owner
+	bloody = istype(human_owner) && human_owner.blood_in_hands
+
+/obj/item/bodypart/arm/generate_icon_key()
+	. = ..()
+	if(bloody)
+		. += "-bloody"
+	return .
+
+/obj/item/bodypart/arm/get_limb_icon(dropped)
+	. = ..()
+	if(bloody)
+		. += image('icons/effects/blood.dmi', bloody_icon, -GLOVES_LAYER, pixel_y = y_offset)
+	return .
```

Comment:
you don't need to do this. the use of . isn't ambiguous

Reply:
I think it's good to do here

Reply:
it's redundant, and since there's only one path through the proc it just looks kinda crummy. If someone gets how . works they'll understand it, and if they don't then they'll be confused by all non explicit uses of it (Initailize() and such)
I guess it depends how you feel about that but I don't think it says anything new here.